group_vars/hosts/telegraf.yml

```diff
@@ -48,3 +48,4 @@ telegraf_plugins:
       interfaces:
         - "bond0"
         - "bond1"
+  - name: exec
```

Comment:
Where are the missing options to tell exec what to execute?

https://github.com/openstack/openstack-ansible-ops/blob/2a1e702209cc713861bae54be2324cd2a59b3121/cluster_metrics/templates/telegraf.conf.j2#L62-L65